### ISSUE TITLE
<amp-facebook> videos missing post content on mobile

### ISSUE THREAD

**Issue by tannerbaum** [ampproject/amphtml#22502]
## What's the issue?

Embedded Facebook videos using <amp-facebook> lose their post contents when on mobile, even when explicitly setting `data-embed-as='post'`. Works perfectly fine when loading the article in the desktop browser, including in responsive mode.  

## How do we reproduce the issue?

This can actually be seen on the official AMP docs, but I also experienced it in my project.

1. Visit https://amp.dev/documentation/examples/components/amp-facebook/?referrer=ampbyexample.com
2. Open browser dev tools and responsive design tools 
3. Pick one of the phones on the device list
4. Scroll down to the video sample embeds and notice the lack of accompanying post text. 

## What browsers are affected?

I tested in Firefox Dev on Mac, but the QA department reported the bug seemingly for all browsers on Android and IOS. 

## Which AMP version is affected?

Version 1905140117570, but the issue was actually reported internally in January so at least that old. 

Responsive mode, or just normal web browser view
![Screenshot 2019-05-24 at 15 51 10](https://user-images.githubusercontent.com/1280597/58353686-2af03180-7e3d-11e9-91fb-04a919536f02.png)

Phone mode
![Screenshot 2019-05-24 at 15 50 45](https://user-images.githubusercontent.com/1280597/58353711-3e030180-7e3d-11e9-82dc-4682a6d2e982.png)

I hope this helps and isn't some dumb thing I am overlooking!

**Comment by nainar** [comment]
Can't be reproduced anymore
